src/sections/Docker-Meshery/docker-extension-CTA.js

```diff
@@ -47,7 +49,7 @@ const DockerExtensionCTA = () => {
     <DockerExtensionCTAWrapper>
       <Container>
         <div className="docker-callout">
-          <img src={Dockerimage} alt="Docker and Meshery" />
+          <StaticImage src={Dockerimage} alt="Docker and Meshery" />
```

Comment:
@avikt18 yeah so there are few svg images for which you used StaticImage so if you can change this we are good to merge

Reply:
Oops!! made a mistake :sweat_smile:. On it :+1: